src/Ryujinx.Horizon/Sdk/Settings/System/NotificationSettings.cs

```diff
@@ -0,0 +1,13 @@
+using System.Runtime.InteropServices;
+
+namespace Ryujinx.Horizon.Sdk.Settings.System
+{
+    [StructLayout(LayoutKind.Sequential, Size = 0x18, Pack = 0x4)]
+    struct NotificationSettings
+    {
+        public uint Flag;
+        public float Volume;
```

Comment:
```suggestion
        public uint Volume;
```

```
    enum NotificationVolume : uint
    {
        Mute,
        Low,
        High,
    }
```

Part of some old WIP code